templates/web/merton/waste/_confirmation_after.html

```diff
@@ -0,0 +1,9 @@
+[% IF first_page == 'medical_condition' %]
+    <p>
+        We will consider your request. We reserve the right to check the information given.
+    </p>
+
+    <p>
+        We will swap your bin for a larger one within 2 weeks, or if your request was unsuccessful we will contact you to let you know.
```

Comment:
I wonder if the above information should be reflected in the email?

Reply:
not sure how to tell it's one in the email, actually, hmm, used `first_page` here. I guess I could check for the extra data being "2::1" but that seems a bit fragile.

Reply:
It could wait until it's specifically requested. Was just a thought, though the person will be contacted if it's not approved anyway.